tests/test_autonomy/test_cli/test_analyse/test_abci/test_specs.py

```diff
@@ -205,8 +192,6 @@ def test_check_all(
         self,
     ) -> None:
         """Test --check-all flag."""
-
-        self._fix_corrupt_file()
```

Comment:
this is a very bad idea! We should never make assumptions about the order in which tests are executed. Each test needs to start from an appropriately clean environment.

Reply:
:+1:  yes, I had noticed this one too